ledger/src/shred.rs

```diff
@@ -474,6 +474,27 @@ impl Shred {
         }
     }
 
+    // Possibly zero pads bytes stored in blockstore.
```

Comment:
nit: This may be repetitive, but could maybe add a comment similar to `bytes_to_store`, something like:
"Should only be used when retrieving shreds from blockstore"

I don't feel too strongly about this one, especially given that the functions are adjacent.